src/types/IJaaSMeetingProps.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import IJitsiMeetingOptions from "./IJitsiMeetingOptions";
+import { JaaSMeeting } from "..";
+
+/**
+ * The type of the React {@code Component} props of {@link JaaSMeeting}.
+ */
+export default interface IJaaSMeetingProps {
+  /**
+   * The id of the component
+   */
+  id: string;
```

Comment:
Why is this needed? I wouldn't know what to set it to.

Reply:
Just to allow more customisation and to make sure the root of the instance is unique, to avoid other react related issues. Will keep it internal, maybe with the possibility of accessing it the same way we do the external API (if it's ok with you).

For the internal implementation, @horymury suggested the use of `generateGUID` from the 8x8meet core, would that be ok?